src/pudl/output/ferc1.py

```diff
@@ -228,6 +228,55 @@ def _out_ferc1__yearly_plants_utilities(
     )
 
 
+## example/draft factory pattern
+specs = [
+    {
+        "name": "out_ferc1__yearly_purchased_power_and_exchanges_sched326",
+        "df1": "core_ferc1__yearly_purchased_power_and_exchanges_sched326",
+        "df2": "core_pudl__assn_ferc1_pudl_utilities",
+        "mg": "utility_id_ferc1",
```

Comment:
Similarly, they all seem to get merged on the same column, so this doesn't need to be part of the specification -- it can just be hard-coded inside the inner factory function definition (unless there are some tables that need a different merge key)